sdks/python/CHANGELOG.md

```
@@ -5,6 +5,23 @@
+- Better documentation all over
+
+### Added
+
+- getPeerConfig to sessions
+
+### Changed
+
+- Summaries now included in toOpenAI and toAnthropic functions for context
+- Length calculation for context object includes summary
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)